mp/src/materialsystem/stdshaders/build_shaders_ags_andrew.bat

```diff
@@ -0,0 +1,27 @@
+@echo off
+setlocal
+@echo=====================================AGS-Renegade Shader Compiler=======================================
```

Comment:
Should we be using the original name of this script? I'd rename to fit our project